fhir-parent/pom.xml

```diff
@@ -64,8 +64,8 @@
         <fhir-persistence-jdbc.index>MINIMAL_JSON</fhir-persistence-jdbc.index>
         <fhir-server-test.index>MINIMAL_JSON</fhir-server-test.index>
         <java.version>1.8</java.version>
-        <fhir-examples.version>4.9.0-SNAPSHOT</fhir-examples.version>
```

Comment:
note that this locks us in on the latest fhir-examples release...if we update fhir-examples then we'll need to come back here and flip it back to SNAPSHOT